src/Arcus.Messaging.Tests.Integration/ServiceBus/ServiceBusClient.cs

```diff
@@ -33,62 +35,108 @@ public ServiceBusClient(KeyRotationConfig configuration, ITestOutputHelper logge
             _logger = logger;
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="entity"></param>
+        /// <returns></returns>
+        public async Task<AccessKeys> GetConnectionStringKeysAsync(ServiceBusEntity entity)
+        {
+            using IServiceBusManagementClient client = await CreateServiceManagementClientAsync();
+
+            return entity switch
+            {
+                ServiceBusEntity.Queue => await client.Queues.ListKeysAsync(
+                    _configuration.ServiceBusNamespace.ResourceGroup,
+                    _configuration.ServiceBusNamespace.Namespace,
+                    _configuration.ServiceBusNamespace.TopicName,
```

Comment:
This should be queue

Reply:
Ah, yes, removed this so it's only for topic. Resolving.